models/AOP_1.ttl

```diff
@@ -7,7 +7,7 @@
     @base <http://www.w3.org/2002/07/owl#> .
 
     <https://noctua.apps.renci.org/model/AOP_1> a owl:Ontology .
-<https://noctua.apps.renci.org/model/AOP_1> <http://purl.org/dc/elements/1.1/title> "Promotion, Hepatocelluar carcinoma"^^xsd:string .
+<https://noctua.apps.renci.org/model/AOP_1> <http://purl.org/dc/elements/1.1/title> "AOP 1"^^xsd:string .
 <https://noctua.apps.renci.org/model/1> <http://geneontology.org/lego/modelstate> "review"^^xsd:string .
```

Comment:
I just realized there's an issue with the model IDs in the modelstate triples on this branch. I'm going to extract the model titles and add them with a SPARQL update instead.